apps/web/src/features/enrollments/components/detail-sections/enrollment-cohort-section.tsx

```
@@ -0,0 +1,191 @@
+				<CardTitle className="flex items-center gap-2">
+					<School className="h-5 w-5" />
+					Cohort Details
+				</CardTitle>
+			</CardHeader>
+			<CardContent className="space-y-6">
+				{/* Product Information */}
+				<div className="grid gap-4 md:grid-cols-2">
+					<div>
+						<label className="font-medium text-muted-foreground text-sm">
+							Linked Product
+						</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)